dubbo-rpc/dubbo-rpc-default/src/main/java/com/alibaba/dubbo/rpc/protocol/dubbo/telnet/ListTelnetHandler.java

```diff
@@ -61,10 +64,16 @@ public String telnet(Channel channel, String message) {
                 if (buf.length() > 0) {
                     buf.append("\r\n");
                 }
+                URL exportUrl = exporter.getInvoker().getUrl();
+                Map<String, String> params = StringUtils.parseQueryString(exportUrl.toFullString());
+                String group = params.get("group");
+                if (StringUtils.isNotEmpty(group)) {
+                    buf.append(group.concat("/"));

```

Comment:
Use `buf.append(group).append("/")` instead, which saves to create an extra String object.